### ISSUE TITLE
<use /> tags 

### ISSUE THREAD

**Issue by KidsKilla** [jhamlet/svg-react-loader#26]
```svg
<svg width='150' height='150' viewBox='0 0 150 150'>
    <defs>
        <rect width='100' height='100' fill='red' id='a'/>
    </defs>
    <rect x='20' y='20' width='100' height='100' fill='green' id='a'/>
    <use xlinkHref='#a'/>
    <rect x='50' y='50' width='100' height='100' fill='black' id='a'/>
</svg>
```

When there is another node (any path/rect/etc) before `<use/>` node, `<use />` node goes below the last node and this breaks picture layering

**Comment by jhamlet** [comment]
@KidsKilla -- Can you provide the webpack/svg-react-loader generated source that this is happening in?

**Comment by KidsKilla** [comment]
Sure. Source is the same as in the topic. 
Generated changed order of `React.createElement('use', { xlinkHref: '#a' }),`
```js
/* 173 */
/***/ function(module, exports, __webpack_require__) {

	'use strict';

	var React = __webpack_require__(2);
	var helpers = __webpack_require__(174)(__webpack_require__(34));

	module.exports = React.createClass({

	    displayName: "Test",

	    getDefaultProps: function getDefaultProps() {
	        return { "width": "150", "height": "150", "viewBox": "0 0 150 150" };
	    },
	    componentDidMount: function componentDidMount() {
	        helpers.applyXmlAttributes(this);
	    },
	    render: function render() {
	        var props = this.props;
	        var children = props.children;

	        return React.createElement(
	            'svg',
	            this.props,
	            React.createElement(
	                'defs',
	                null,
	                React.createElement('rect', { width: '100', height: '100', fill: 'red', id: 'a' })
	            ),
	            React.createElement('rect', { x: '20', y: '20', width: '100', height: '100', fill: 'green', id: 'a' }),
	            React.createElement('rect', { x: '50', y: '50', width: '100', height: '100', fill: 'black', id: 'a' }),
	            React.createElement('use', { xlinkHref: '#a' }),
	            React.Children.map(children, function (c) {
	                return c;
	            })
	        );
	    }
		});

/***/ },
```

**Comment by KidsKilla** [comment]
Is there any info when to expect fix for this?

**Comment by jhamlet** [comment]
I'm aiming for [v0.4.0](https://github.com/jhamlet/svg-react-loader/milestone/5)

Do you have a sample SVG file that I can reproduce this with?

**Comment by KidsKilla** [comment]
@jhamlet 

> I'm aiming for v0.4.0
Cool, thanks!

> Do you have a sample SVG file that I can reproduce this with?
The one that is in the topic. 
It should have black square above the others, but now there's red one.

**Comment by jhamlet** [comment]
The issue was with `xml2js` not keeping its children in order. This should be fixed with v0.4.0-beta.2
